<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsVectorLayerLoadStyleDialog</class>
 <widget class="QDialog" name="QgsVectorLayerLoadStyleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>597</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Load Layer Style</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1" colspan="5">
    <widget class="QListView" name="mStyleCategoriesListView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mModeLabel">
     <property name="text">
      <string>Load style</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close|QDialogButtonBox::Help|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Categories</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mFileLabel">
     <property name="text">
      <string>File</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QgsFileWidget" name="mFileWidget"/>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QWidget" name="mFromDbWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Styles related to the layer</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTableWidget" name="mRelatedTable">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Other styles on the database</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTableWidget" name="mOthersTable">
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QComboBox" name="mStyleTypeComboBox"/>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="mInvertSelectionButton">
     <property name="text">
      <string>Invert Selection</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="mDeselectAllButton">
     <property name="text">
      <string>Deselect All</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="mSelectAllButton">
     <property name="text">
      <string>Select All</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsFileWidget</class>
   <extends>QWidget</extends>
   <header>qgsfilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mStyleTypeComboBox</tabstop>
  <tabstop>mRelatedTable</tabstop>
  <tabstop>mOthersTable</tabstop>
  <tabstop>mStyleCategoriesListView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
